vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -155,6 +155,13 @@ The code below adds two components to the page:
 
 As you explore the dashboard, you may notice that the current layout could be further enhanced. The charts may appear cramped, while the text component has ample unused space. The next section explains how to configure the layout and arrange the components.
 
+!!! tip "An introduction to Vizro-AI"
+
+    In the example above, the code to create the line graph was generated using [Vizro-AI](https://vizro.readthedocs.io/projects/vizro-ai/en/latest/pages/get-started/quickstart/). Vizro-AI enables you to use English, or other languages, to create create interactive charts with [Plotly](https://plotly.com/python/) by simplifying the process through use of a large language model. In essence, Vizro-AI generates code from natural language instructions so that you can add it into a Vizro dashboard, such as in the example above.
```

Comment:
@maxschulz-COL @nadijagraca I've added a small tip box here to explain that this code matches the example at the start of the Vizro-AI docs (to highlight that Vizro-AI exists and show how it can be used to make chart creation easier when you're building a dashboard). 

It's not supposed to be a complete explanation of how to use Vizro-AI at this point, so I just dropped the call to plotly in rather than the full function...I suggest that I add a link to the upcoming docs that @nadijagraca has under review.

However, because this approach is slightly different I wonder if it could confuse people. I don't think it will because it is more of a nudge to go look at Vizro-AI rather than an explainer, but WDYT?